R/loadMAVISp.R

```diff
@@ -0,0 +1,76 @@
+#' loadMAVISp
+#' 
+#' This function loads the MAVISp database from the directory specified by the user. 
+#' 
+#' @param mavispDB path to the MAVISp database
+#' @param proteins_of_interest vector containing specific proteins of interest in HUGO format
+#' @importFrom stringr str_c str_split_i
+#' @importFrom purrr map
+#' @importFrom readr read_csv
+#' @importFrom tibble as_tibble_col
+#' @importFrom dplyr pull filter
+#' @importFrom rlang set_names
+#' @importFrom withr with_options
+#' @return returns a list of tibbles each containing the MAVISp entry of one protein
+#' @export
+#' @examples
+#' 
+#' mavisp_data <- loadMAVISp(mavispDB = "/data/raw_data/computational_data/mavisp_database/biorxiv_v4_17112023",
+#'           proteins_of_interest = c('NQO1','TP53'))
+#' 
+#' mavisp_data <- loadMAVISp(mavispDB = "/data/raw_data/computational_data/mavisp_database/biorxiv_v4_17112023") 
```

Comment:
we should consider on how to do these examples - examples should be runnable. We can either include a dataset with the package, or consider if we want to make the whole MAVISp dataset as a data package from BioC which would also be useful in general to the R community